modules/local/utils.nf

```diff
@@ -88,7 +90,11 @@ process CHANGE_READS {
     export to_accession=\$(cat help_file | cut -f2)
     echo "\${from_accession} --> \${to_accession}"
 
-    change_reads.py --reads ${reads} -f \${from_accession} -t \${to_accession} --change_dots_to_underscores
+    gunzip ${reads}
```

Comment:
maybe return sed command?

Reply:
we used `sed -i "s/_/\./" $F` in our commands before to change dots to underscores and can do the same with accessions